.github/workflows/build.yml

```diff
@@ -17,6 +17,9 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Setup Compiler
+      run: |
+        sudo apt-get -y install gfortran
```

Comment:
In the future this may become a more general "Setup environment" step.

Reply:
I expect you're right - do you want me to rename it so now or shall we leave it as "Compiler"?

Reply:
Leave it. It is correct at the moment.